bazel/proxy_wasm_cpp_sdk.patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/proxy_wasm_api.h b/proxy_wasm_api.h
+index 166b49c..b44637c 100644
+--- a/proxy_wasm_api.h
++++ b/proxy_wasm_api.h
+@@ -1207,8 +1207,9 @@ struct SimpleHistogram {
+ template <typename... Tags> struct Counter : public MetricBase {
+   static Counter<Tags...> *New(std::string_view name, MetricTagDescriptor<Tags>... fieldnames);
+ 
+-  Counter<Tags...>(std::string_view name, MetricTagDescriptor<Tags>... descriptors)
+-      : Counter<Tags...>(std::string(name), std::vector<MetricTag>({toMetricTag(descriptors)...})) {
++  template <typename... T>
```

Comment:
Is proxy_wasm_api incompatible with C++20?

Reply:
No, this is just some helper AFAICT which is only used in one place in a test.